### DISCUSSION TITLE
Tahoma2d symmetry tool

### DISCUSSION THREAD

**Discussion by Dixxi0509** [tahoma2d/tahoma2d#1158] [Ideas]
I had a idea where tahoma2d has a symmetry tool/ruler to make easier for people who draw and animate on the software. And yeah I know there horizonal and vertical tools with copy and paste but I think this would make it easier! (This can go for opentoonz too!) 

This is just a idea

**Comment by manongjohn** [comment]
There is a new Symmetry Tool feature which will be available in the next 1.4 release.  Download the latest nightly and try it out.

**Comment by manongjohn** [comment]
Can read up on Symmetry Tool in #1131

**Comment by Dixxi0509** [comment]
When would 1.4 come out
